Src/Microsoft.Dynamic/Microsoft.Dynamic.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project>
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Yes, the current implementation was causing issues (although at this point I don't remember what exactly).